### PR TITLE
Mhr Organization Name Lookup

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#16969

*Description of changes:*
* Updated Search field to include auto complete for MHR Organization Name
* Disable help text for Search Instances of Business Lookup

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/search/SearchBar.vue [hunk 1]
```diff
@@ -69,13 +69,53 @@
...
+                @search-value="setSearchValue"
+                @searching="loadingSearchResults = $event"
+              />
+            </v-card>
+          </v-col>
+
+          <v-col v-else-if="isMhrOrgSearch">
+            <v-text-field
+                filled
+                id="txt-mhr-org-name-"
```
Comment (dimak1) [inline ppr-ui/src/components/search/SearchBar.vue]: Small typo here.

**Review by dimak1 — APPROVED** [review]

File: ppr-ui/src/components/search/SearchBar.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/search/SearchBar.vue]: Thank you, will fix!

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1420-0uo7gjr6.web.app

**Review by doug-lovett — APPROVED** [review]
Tres bon!

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1420-0uo7gjr6.web.app

**Pull request merged**